src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -2458,11 +2554,94 @@ Note: `aparse()` is case-insensitive.
         Use `string()` to facet by a floating-point value without losing decimal places.
 
         ```sql
-        FROM PageView SELECT count(*) FACET string(tuple(asnLatitude, asnLongitude), precision: 2) 
+        FROM PageView SELECT count(*) FACET string(tuple(asnLatitude, asnLongitude), precision: 2)
+        ```
+      </Collapser>
+    </CollapserGroup>
+  </Collapser>
+
+  <Collapser
+    className="freq-link"
+    id="func-substring"
+    title={<InlineCode>substring(str, start [, length])</InlineCode>}
+  >
+    Use the substring() function to extract a portion of a string.
+
+    Arguments:
+    * str - the string from which to extract a substring.
+      * This can be anything that evaluates to a string, including a literal string in quotes, a queried string attribute, a function that returns a string, or even a subquery that returns a single string value.
+      * If this argument evaluates to null, the substring function will return null.
+    * start - the position within str from which to begin the extraction.
+      * The first character in str is position 0.
+      * A negative value will find the position relative to the end of str, with the last character of the string being position -1.
+      * If start is larger or equal to the length of str, the substring function will return an empty string.
+      * If start is negative, and its absolute value is larger than the length of str, the extracted substring will begin at position 0.
+    * length - the length, or number of characters, of the substring to extract from str.
+      * _Optional_ - if length is not provided, all characters in str after the resolved start position will be included.
+
+    <CollapserGroup>
+      <Collapser title="Use substring to extract various parts of a string">
+        This query returns various parts of session value.
```

Comment:
of _the_ session value

Reply:
I think you could remove "various" from both lines without affecting the meaning.